tests/integration/run.bats

```diff
@@ -241,3 +256,19 @@ function runnc_run() {
 	[ "$status" -eq 0 ]
 	[[ "$output" == *"--mem=1024"* ]]
 }
+
+@test "oom_adjust" {
+	setup_test "node"
+	local name="test-nabla-oom-adjust"
+
+	config_mod '.process.args |= .+ ["node.nabla", "/hello/app.js"]'
+	config_mod '.process.OOMScoreAdj |= .+ 104'
```

Comment:
Let's try making this `oomScoreAdj` according to the spec json. see if it will work as well?